src/processes.c

```diff
@@ -761,13 +775,20 @@ static void ps_submit_proc_list(procstat_t *ps) {
     plugin_dispatch_values(&vl);
   }
 
-  if (ps->num_fd > 0) {
+ if (ps->num_fd > 0) {
```

Comment:
This change also unneeded.

Reply:
I'd like to note that *clang-format* has the final say in this. I'd say it would undo this change.